custom-recipes/nlp-analysis-ontology-tagging/recipe.py

```diff
@@ -4,12 +4,20 @@
 from ontology_tagger import Tagger
 
 
-settings = DkuConfigLoadingOntologyTagging().load_settings()
+dku_config = DkuConfigLoadingOntologyTagging()
+settings = dku_config.load_settings()
 text_dataframe = settings.text_input.get_dataframe(infer_with_pandas=False)
 ontology_dataframe = settings.ontology_input.get_dataframe(
     columns=settings.ontology_columns, infer_with_pandas=False
 )
 
+languages = (
```

Comment:
This should be done by DkuConfigLoadingOntologyTagging

Reply:
it can't, as we are loading the text dataframe in recipe.py

Reply:
ok, let's keep it then